bootstrapper/initproto/init.proto

```diff
@@ -16,8 +16,6 @@ message InitRequest {
   string storage_uri = 2;
   // MeasurementSalt is a salt used to generate the clusterID for the initial bootstrapping node.
   bytes measurement_salt = 3;
-  // CloudServiceAccountUri is an URI encoding the cloud service account.
-  string cloud_service_account_uri = 4;
```

Comment:
Until we decide that we want to completely remove the lines of removed entries, I'd prefer:
```
// string cloud_service_account_uri = 4; removed
```